### ISSUE TITLE
Stroking cock blocked by urethra size

### ISSUE THREAD

**Issue by A-Master-of-Puppets** [Innoxia/liliths-throne-public#316]
If a character doesn't have a penetrable urethra, the action to stroke their cock is not available.

**Comment by Innoxia** [comment]
Yeah, I need to add functionality to have a sex action be between two penetration or two orifice types, instead of always 1 penetration + 1 orifice.

**Comment by A-Master-of-Puppets** [comment]
That or add "hand" as a type of orifice. Strictly speaking it isn't, but strictly speaking neither is cleavage (ass or breast) or thighs.

**Comment by A-Master-of-Puppets** [comment]
Allowing pen/pen or orifice/orifice would probably be better though; would allow for more logical pussy licking, and for scissoring to be added easily.

**Comment by Innoxia** [comment]
I've fixed this in 0.2.3.1 (through a bit of a hack in `SexActionInterface.java`'s `toResponse()` method, near line 242).

I will implement pen/pen or orifice/orifice actions as soon as I can though, as they will be useful in the future.

**Comment by A-Master-of-Puppets** [comment]
If it's just fixed by a temporary hack, maybe the issue should stay open until the proper fix is made? Just a thought.

**Comment by Innoxia** [comment]
I've made a new issue for this: #333
